### PR TITLE
Fix functions interpreting string args as booleans

### PR DESCRIPTION
**Pull request by prachirp**
Link GoogleContainerTools/kpt#823

### CODE REVIEW

**Comment by prachirp** [comment]
cc @frankfarzan , @Shell32-Natsu

**Comment by Shell32-Natsu** [comment]
Has the issue gone?

**Comment by prachirp** [comment]
Yes, tried it using gcr.io/kpt-functions/istioctl-analyze:dev and it ran without issue.

File: functions/ts/src/helm_template.ts [hunk 1]
```diff
@@ -50,8 +50,8 @@ export async function helmTemplate(configs: Configs) {
 
 function readArguments(configs: Configs) {
   const args: string[] = [];
-  let nameArg;
-  let pathArg;
+  let nameArg = '';
```
[the remark this answers is not in the document]
Comment (prachirp) [reply functions/ts/src/helm_template.ts]: Done

File: functions/ts/src/helm_template.ts [hunk 2]
```diff
@@ -50,8 +50,8 @@ export async function helmTemplate(configs: Configs) {
 
 function readArguments(configs: Configs) {
   const args: string[] = [];
-  let nameArg;
-  let pathArg;
+  let nameArg: string | undefined = undefined;
```
Comment (frankfarzan) [inline functions/ts/src/helm_template.ts:53]: default is undefined:

let nameArg: string | undefined;

**Pull request merged**